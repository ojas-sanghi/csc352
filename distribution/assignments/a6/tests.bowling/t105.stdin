10
6
4
7
3
10
9
1
7
3
10
9
1
5
5
8
2
8
