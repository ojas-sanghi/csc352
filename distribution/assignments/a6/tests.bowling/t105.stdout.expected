Frame  1: (10/ -) 20  20 (Strike)
Frame  2: ( 6/ 4) 17  37 ( Spare)
Frame  3: ( 7/ 3) 20  57 ( Spare)
Frame  4: (10/ -) 20  77 (Strike)
Frame  5: ( 9/ 1) 17  94 ( Spare)
Frame  6: ( 7/ 3) 20 114 ( Spare)
Frame  7: (10/ -) 20 134 (Strike)
Frame  8: ( 9/ 1) 15 149 ( Spare)
Frame  9: ( 5/ 5) 18 167 ( Spare)
Frame 10: ( 8/ 2) 18 185 ( Spare) (+8)
